<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stereo</class>
 <widget class="QWidget" name="Stereo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>274</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QCheckBox" name="stereoShowDisparity">
       <property name="text">
        <string>show disparity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stereoColor"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QCheckBox" name="stereoUseForReco">
       <property name="toolTip">
        <string>the recognition is not only done by markers; the height profile of a person is searched inside the disparity map</string>
       </property>
       <property name="text">
        <string>use for recognition</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stereoDispAlgo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="stereoUseForHeight">
       <property name="toolTip">
        <string>enable, if the disparity should be used to measure the position in 3D space and use it for height measurement</string>
       </property>
       <property name="text">
        <string>use for height/pos measurement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="stereoUseForHeightEver">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>ever</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="stereoUseForExport">
       <property name="toolTip">
        <string>the calculated 3D data is directly used for trajectory export; values in the calibration tab are ignored (for uneven videos)</string>
       </property>
       <property name="text">
        <string>use for trajectory export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="stereoUseCalibrationCenter">
       <property name="toolTip">
        <string>while exporting pointGrey data directly, the coordinate center of calibration tab is used</string>
       </property>
       <property name="text">
        <string>use calib center</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>stereo mask size: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>edge mask size: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>min disparity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="stereoMaskSize">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>23</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="edgeMaskSize">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>11</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="maxDisparity">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="minDisparity">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>max disparity</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="useEdge">
       <property name="text">
        <string>use</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>opacity</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="opacity">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="hideWrong">
       <property name="text">
        <string>hide</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="stereoExport">
     <property name="text">
      <string>export point cloud</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
